src/app/components/admin/admin-evaluations/admin-evaluations.component.ts

```diff
@@ -27,10 +27,8 @@ import { UserDataService } from 'src/app/data/user/user-data.service';
 })
 export class AdminEvaluationsComponent implements OnInit, OnDestroy {
   @Input() evaluationList: Evaluation[];
-  @Input() pageSize: number;
-  @Input() pageIndex: number;
-  @Output() sortChange = new EventEmitter<Sort>();
-  @Output() pageChange = new EventEmitter<PageEvent>();
+  pageIndex: number = 0;
+  pageSize: number = 10;
   filterControl: UntypedFormControl = this.evaluationDataService.filterControl;

```

Comment:
Because the filter control is set to the evaluationDataService.filterControl, it is using the filtering from evaluationDataService and NOT the filtering in this component.  If you comment out lines 203 - 209, the filtering still happens.

The filterControl should be set to new UntypedFormControl() and it should be observed for changes by adding code similar to this in the constructor:

    this.filterControl.valueChanges
      .pipe(takeUntil(this.unsubscribe$))
      .subscribe((term) => {
        this.filterString = term.trim().toLowerCase();
        this.applyFilter();
      });